test/Integration/SqlTriggerBindingIntegrationTests.cs

```diff
@@ -588,11 +588,13 @@ public void UnsupportedDatabaseThrows(SupportedLanguages lang)
         /// <summary>
         /// Tests that when a user function throws an exception we'll retry executing that function once the lease timeout expires
         /// </summary>
-        [Fact]
-        public async Task FunctionExceptionsCauseRetry()
+        [Theory]
+        [SqlInlineData()]
+        [UnsupportedLanguages(SupportedLanguages.JavaScript, SupportedLanguages.Python, SupportedLanguages.PowerShell, SupportedLanguages.CSharpscript)] //Static state threwException is only valid for C# and Java.
```

Comment:
What does this mean?

Reply:
Lucy told me that she intentionally skipped this since the static variables isn't valid for those languages, now that I think about it I can add a variable and add the test.

Reply:
The FunctionExceptionsCauseRetry test relies on the static [threwException](https://github.com/Azure/azure-functions-sql-extension/blob/33ee38c3f1a578093cf7612649ead737b20f8120/test/Integration/test-csharp/TriggerWithException.cs#L14) value in the function but I don't think there is a way to do this in JS, Python, PS.

Reply:
Hm, ok this is fine for now then. Hopefully can figure out a way to do this with a bit more digging in the future but not a huge concern. @MaddyDev could you file an issue to track looking into this?